sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AzureADGraphClient.java

```diff
@@ -12,6 +12,8 @@
 import com.microsoft.aad.msal4j.OnBehalfOfParameters;
 import com.microsoft.aad.msal4j.UserAssertion;
 import com.nimbusds.oauth2.sdk.http.HTTPResponse;
+import java.util.Arrays;
+import java.util.stream.Stream;
```

Comment:
https://github.com/Azure/azure-sdk-for-java/pull/17325/files
After this(jialin's) PR merged, we will not use `AzureADGraphClient` any more. 

So my suggestion is to abandon this PR and learn jialin's PR. 

Please confirm that in jialin's PR, whether `scope/role` Authority work well with multiple resourceUri and multiple tenant. 🙏

Reply:
Okay,thank you for your suggestion,
`confirm that in jialin's PR, whether scope/role Authority work well with multiple resourceUri and multiple tenant. `
I will confirm it.